{
  "code": "def find(num1, num2):\n    try:\n        return num1 / num2\n    except ZeroDivisionError:\n        return \"Error: Division by zero is not allowed\"\n",
  "results": {
    "assert find(10,3) == 3": {
      "result": false,
      "time": 1.264787197113037,
      "error": null
    },
    "assert find(4,2) == 2": {
      "result": true,
      "time": 0.7817091941833496,
      "error": null
    },
    "assert find(20,5) == 4": {
      "result": true,
      "time": 0.405897855758667,
      "error": null
    }
  },
  "passed": false,
  "accuracy": 0.6666666666666666,
  "general_error": null,
  "has_testcase_error": false
}